python/common/org/python/types/Str.java

```diff
@@ -543,10 +543,66 @@ else if (other instanceof org.python.types.Bool) {
     }
 
     @org.python.Method(
-        __doc__ = ""
-    )
-    public org.python.Object center() {
-        throw new org.python.exceptions.NotImplementedError("center() has not been implemented.");
+        __doc__ = "",
+		args = {"width"},
+		default_args = {"charToFill"}
+    )
+    public org.python.Object center(org.python.Object width, org.python.Object charToFill) {
+		if(this instanceof org.python.types.Str)
```

Comment:
I thought about this but I have very little knowledge about how it's called internally. So kept the check to be safe